functorch/_src/aot_autograd.py

```diff
@@ -369,13 +369,6 @@ def add_dupe_args(args):
     joint_forward_backward = create_joint_forward_backward(lambda *args: flat_fn(*add_dupe_args(args)))
 
     out = flat_fn(*flat_args)
-    # Collect info on which output tensors require gradients,
-    # so we can mark them properly in the returned autograd.Function
-    _flat_outs_not_requiring_grad, _ = pytree.tree_flatten(
```

Comment:
oh did this break something that I should look into?

Reply:
no, i fixed it